internal/infrastructure/services/task_test.go

```diff
@@ -588,6 +600,10 @@ func TestFailTTL(t *testing.T) {
 		taskID, err := service.Create("owner", "test", 3, 1, "", 0, 5, "")
 		assert.Nil(t, err)
 
+		sheduledTaskID, err := schedulerRepo.NextInQueue(ctx, "test")
```

Comment:
```suggestion
		scheduledTaskID, err := schedulerRepo.NextInQueue(ctx, "test")
```

Ainsi que les 15 copié / collé qui suivent.